packages/extra/test/reference/DayOfMonthTest.js

```diff
@@ -80,11 +80,16 @@ describe('org.threeten.extra.TestDayOfMonth', () => {
     //-----------------------------------------------------------------------
     describe('now()', () => {
         it('test_now', () => {
-            let test = DayOfMonth.now();
-            if (LocalDate.now().dayOfMonth() !== test.value()) {
-                test = DayOfMonth.now();
+            let expected;
+            let actual;
+            for (let i = 0; i < 100; i++) {
+                expected = LocalDate.now().dayOfMonth();
+                actual = DayOfMonth.now().value();
+                if (expected === actual) {
+                    break;
+                }
             }
-            assertEquals(LocalDate.now().dayOfMonth(), test.value());
+            assertEquals(expected, actual);
```

Comment:
Oh, it can pass if expected and actual are undefined (after 100 unsuccessful retries) :/
I'll need to fix this once again.

Reply:
good catch